loadtest/utils.go

```diff
@@ -90,6 +90,8 @@ func nextPowerOf2(val int) int {
 	return val
 }
 
+// RandomFutureTime returns a random Unix timestamp, in milliseconds, in the interval
+// [now+deltaStart, now+maxUntil]
```

Comment:
The comment is actually correct, thanks! But it surfaced a bug in this function: if `maxUntil` is smaller than `deltaStart` (let's say, you call `RandomFutureTime(10*time.Second, 9*time.Second)`, then `rand.Int63n` panics because it gets a negative number). We should fix the function so that the interval is actually `[now+deltaStart, now+deltaStart+maxUntil]`, and add unit testing for this function.

Reply:
Done.